pkg/http/server.go

```diff
@@ -96,6 +98,21 @@ func (s *server) useCors(next fasthttp.RequestHandler) fasthttp.RequestHandler {
 	return corsHandler.CorsMiddleware(next)
 }
 
+func useAPIAuthentication(next fasthttp.RequestHandler) fasthttp.RequestHandler {
```

Comment:
How does this gets plugged into request pipeline?

Reply:
huh, did this for gRPC and not HTTP. fixed now.